### PR TITLE
Releasing release.yaml v0.21.0

### PR DESCRIPTION
**Pull request by pradeepitm12**
Signed-off-by: Pradeep Kumar <pradkuma@redhat.com>

### CODE REVIEW

**Comment by pradeepitm12** [comment]
cc: @chmouel @piyush-garg

File: openshift/release/tektoncd-pipeline-v0.21.0.yaml [hunk 1]
```diff
@@ -0,0 +1,2079 @@
...
+        app: tekton-pipelines-controller
+        version: v0.21.0
+    spec:
+      serviceAccountName: tekton-pipelines-controller
+      containers:
+      - name: tekton-pipelines-controller
+        image: quay.io/openshift-pipeline/tektoncd-pipeline-controller:v0.21.0
+        args: [
+          # Version, to be replace at release time
+          "-version", "devel",
```
Comment (piyush-garg) [inline openshift/release/tektoncd-pipeline-v0.21.0.yaml:1657]: This needs to be fixed

File: openshift/release/tektoncd-pipeline-v0.21.0.yaml [hunk 2]
```diff
@@ -0,0 +1,2079 @@
...
+          value: /etc/ssl/certs
+        - name: METRICS_DOMAIN
+          value: tekton.dev/pipeline
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - all
+          # User 65532 is the distroless nonroot user ID
+          runAsGroup: 65532
```
Comment (piyush-garg) [inline openshift/release/tektoncd-pipeline-v0.21.0.yaml:1714]: Should we remove these also, we remove runAsUser before cc @chmouel

**Review by vdemeester — APPROVED** [review]

File: openshift/release/tektoncd-pipeline-v0.21.0.yaml (hunk 2 above)
Comment (chmouel) [reply openshift/release/tektoncd-pipeline-v0.21.0.yaml:1714]: yup.

**Pull request merged**